src/app/shared/pipes/timestamp.pipe.ts

```diff
@@ -0,0 +1,18 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'timestamp'
+})
+export class TimestampPipe implements PipeTransform {
+  transform(timestampInSeconds: number,): string {
```

Comment:
I know this method is public, due to there being no access modifier, and it being the implementation of the interface, but could we please add access modifiers explicitely?

Reply:
Access modifier is made explicit and the extra comma has been removed.